roles/kube-burner/files/metrics-aggregated.yaml

```diff
@@ -2,10 +2,13 @@ metrics:
   - query: (sum(irate(container_cpu_usage_seconds_total{name!="",container!="POD",namespace=~"openshift-(etcd|oauth-apiserver|sdn|ovn-kubernetes|.*apiserver|authentication|.*controller-manager|.*scheduler)"}[2m]) * 100) by (container, pod, namespace, node) and on (node) kube_node_role{role="master"}) > 0
     metricName: containerCPU-Masters
 
+  - query: (sum(irate(container_cpu_usage_seconds_total{pod!="",container="prometheus",namespace="openshift-monitoring"}[2m]) * 100) by (pod, node) and on (node) kube_node_role{role="infra"}) > 0
```

Comment:
I think is better to aggregate by (container, pod, namespace, node) rather than only (pod, node), it will increase the number of documents, however as there's only two prometheus replicas, that increase won't be too much

Reply:
Fixed it, thanks.